JASP-Engine/JASP/R/anovarepeatedmeasures.R

```diff
@@ -2537,10 +2540,13 @@ AnovaRepeatedMeasures <- function(dataset=NULL, options, perform="run", callback
 	list(result=descriptivesPlotList, status=status, stateDescriptivesPlot=stateDescriptivesPlot)
 }
 
-.rmAnovaSimpleEffects <- function(dataset, options, perform, fullModel, fullAnovaTableWithin, fullAnovaTableBetween, status, singular, stateSimpleEffects) {
+.rmAnovaSimpleEffects <- function(dataset, options, perform, fullModel, fullAnovaTableWithin, 
+                                  fullAnovaTableBetween, status, singular, stateSimpleEffects) {
   
-  if (identical(options$simpleFactor, "") | identical(options$moderatorFactorOne, ""))
+  if (identical(options$simpleFactor, "") | identical(options$moderatorFactorOne, "")) {
     return (list(result=NULL, status=status))
+  }
+  
```

Comment:
Perhaps here is a good point to add the check for "Type 1" and return

Reply:
I tried that, but it causes the whole ANOVA to crash - all the more reason for a rewrite!